### ISSUE TITLE
printf: missing unicode digit for \U

### ISSUE THREAD

**Issue by kkopachev** [xdissent/ievms#265]
```
Waiting for IE11 - Win7 to be available for guestcontrol...
bash: line 38: printf: missing unicode digit for \U
Copying IE11-Windows6.1-x86-en-us.exe to C:\Users\IEUser\Desktop\IE11-Windows6.1-x86-en-us.exe
```

**Comment by xdissent** [comment]
Can you tell me your host system and bash version?

**Comment by kkopachev** [comment]
Ubuntu 15.04 
```
bash --version
GNU bash, version 4.3.30(1)-release (x86_64-pc-linux-gnu)
Copyright (C) 2013 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>

This is free software; you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.
```

**Comment by xdissent** [comment]
Ah, one of the GNU extensions is unicode escape code interpreting in `printf` apparently. I should handle that better. Patch forthcoming!

**Comment by xdissent** [comment]
Can you try https://raw.githubusercontent.com/xdissent/ievms/fix-log/ievms.sh for me?

**Comment by kkopachev** [comment]
Looks good!
```
Waiting for IE11 - Win7 to be available for guestcontrol...
Waiting for IE11 - Win7 to be available for guestcontrol...
Waiting for IE11 - Win7 to be available for guestcontrol...
Copying IE11-Windows6.1-x86-en-us.exe to C:\Users\IEUser\Desktop\IE11-Windows6.1-x86-en-us.exe
        1 file(s) copied.
Installing IE
```

**Comment by xdissent** [comment]
Fixed in v0.3.1! Thanks for catching that!
